api/notifications.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"github.com/wtg/shuttletracker/model"
+	"net/http"
+	"strings"
+	"encoding/json"
+)
+
+// AdminMessageHandler handles the retrieval of the current administrator message
+func (api *API) AdminMessageHandler(w http.ResponseWriter, r *http.Request) {
+	message, err := api.db.GetCurrentMessage()
+
+	if err != nil {
+		http.Error(w,err.Error(), http.StatusInternalServerError)
+	}
+	WriteJSON(w, message)
+}
+
+// SetAdminMessage allows the user to set an alert message that will display to all users who visit the page
+func (api *API) SetAdminMessage(w http.ResponseWriter, r *http.Request){
```

Comment:
This function should validate the length of the provided message (character limit) and return an HTTP 400-something if it's too long.